epp-client/src/connection/client.rs

```diff
@@ -49,14 +50,15 @@
 use std::time::SystemTime;
 use std::{error::Error, fmt::Debug};
 
+use crate::common::{EppObject, NoExtension};
 use crate::config::EppClientConfig;
 use crate::connection::registry::{epp_connect, EppConnection};
 use crate::error;
 use crate::hello::{EppGreeting, EppHello};
```

Comment:
Could you also rename `EppHello` and `EppGreeting` to `Hello` and `Greeting` to be more in line with the renaming of the other request and response types?